packages/components/src/internal/app/utils.ts

```diff
@@ -115,6 +115,14 @@ export function initWebSocketListeners(
     }
 }
 
+export function userCanManagePicklists(user: User): boolean {
+    return hasAllPermissions(user, ['org.labkey.api.lists.permissions.ManagePicklistsPermission']);
+}
+
+export function userCanDeletePublicPicklists(user: User): boolean {
+    return hasAllPermissions(user, [PermissionTypes.ApplicationAdmin]);
```

Comment:
I see. That seem a little odd to me that it wouldn't be based on the DesignList permissions. Would that also be the case in LKS (i.e. that someone needs to be an admin to delete public pick lists?). As you said though, I think just admin perm would suffice instead of needing ApplicationAdmin.